### ISSUE TITLE
[Feature Request] Add "Open in default browser" option

### ISSUE THREAD

**Issue by Darthagnon** [yang991178/fluent-reader#453]
### Is your feature request related to a problem? Please describe.
When reading feed articles, often the content is truncated, behind a paywall (news websites), full of advertising/cookie banners, or unavailable (e.g. YouTube videos). This makes them unreadable in the article view and also in the internal browser, which is already a rather small window.

### The simple solution
![1gP5MzU43O](https://user-images.githubusercontent.com/20792748/186236004-55926917-5a31-4841-b100-af4c72bb014b.png)
Add a setting that makes this button "Open in default browser", instead of the internal browser. I'd suggest even ditching the internal browser and making this the only option, as it will always be inferior to reading in a full desktop browser, but that might be a step too far.

### The ideal (but impossible) solution
Adding built-in [uBlock Origin](https://github.com/gorhill/uBlock), a bunch of [filterlists](https://filterlists.com/), Bypass Paywalls Clean (magnolia1234/bypass-paywalls-chrome-clean on GitLab), [I Don't Care About Cookies](https://www.i-dont-care-about-cookies.eu/) and some sort of Reader View (e.g. [JustRead](https://justread.link/), [ReaderMode](https://readermode.io/), [Reader View](https://github.com/rNeomy/reader-view)) to the internal browser. Or somehow extracting fulltext articles (would require a lot of per-site adjustments). Much better solution just to make links open in the default external browser, which will already have all the necessary filtering extensions.

### Describe alternatives you've considered
My main RSS reader is currently [FeedBro](https://nodetics.com/feedbro/)

### Related issues
#442, #444, #432, #410, #266

**Comment by yang991178** [comment]
FYI, you can set certain feeds to open directly in the browser in settings, or press `b` to open in browser from the article view.

**Comment by jadedgnome** [comment]
yes, can get an option to manually choose what browser to launch and set it's location?
